DependencyInjection/Compiler/TranslationResourceFilesPass.php

```diff
@@ -17,16 +17,16 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        if (Kernel::VERSION_ID < 20700) {
-            $translationFiles = $this->getTranslationFilesFromOlderSymfonyVersions($container);
-        } else {
-            $translationFiles = $this->getTranslationFiles($container);
+        $translationFiles = $this->getTranslationFilesFromAddResourceCalls($container);
+
+        if (Kernel::VERSION_ID >= 20700) {
```

Comment:
I even suggest calling this method all the time:

- it is possible to have different versions of FrameworkBundle, HttpKernel and Translator installed, and ``Kernel::VERSION_ID`` detects the HttpKernel version here, while we deal with FrameworkBundle and Translator
- the method already returns an empty array in case the option is not defined, which will be compatible with older versions of Symfony